pymatgen/entries/tests/test_correction_calculator.py

```diff
@@ -57,28 +58,28 @@ def setUp(self):
         }
 
         self.no_uncertainties_corrections = {
-            "oxide": (-0.744, 0.0193),
-            "peroxide": (-0.466, 0.0478),
-            "superoxide": (-0.155, 0.0529),
-            "sulfide": (-0.638, 0.0375),
-            "F": (-0.436, 0.02),
-            "Cl": (-0.517, 0.0258),
-            "Br": (-0.436, 0.0285),
-            "I": (-0.315, 0.0338),
-            "N": (-0.357, 0.0599),
-            "Se": (-0.361, 0.053),
-            "Si": (-0.439, 0.1017),
-            "Sb": (-0.286, 0.0753),
-            "Te": (-0.438, 0.063),
-            "V": (-1.66, 0.0759),
-            "Cr": (-1.85, 0.0679),
-            "Mn": (-1.589, 0.0689),
-            "Fe": (-2.165, 0.0606),
-            "Co": (-1.563, 0.0771),
-            "Ni": (-2.516, 0.0961),
-            "W": (-4.574, 0.1199),
-            "Mo": (-3.05, 0.1085),
-            "H": (-0.151, 0.0323),
+            'oxide': (-0.744, 0.0194),
+            'peroxide': (-0.466, 0.0481),
+            'superoxide': (-0.155, 0.0534),
+            'sulfide': (-0.638, 0.0378),
+            'F': (-0.436, 0.0202),
+            'Cl': (-0.517, 0.026),
+            'Br': (-0.436, 0.0287),
+            'I': (-0.29, 0.032),
+            'N': (-0.357, 0.0604),
+            'Se': (-0.361, 0.0534),
+            'Si': (-0.439, 0.1025),
+            'Sb': (-0.286, 0.0759),
+            'Te': (-0.457, 0.0513),
+            'V': (-1.66, 0.0765),
+            'Cr': (-1.85, 0.0684),
+            'Mn': (-1.589, 0.0695),
+            'Fe': (-2.165, 0.061),
+            'Co': (-1.563, 0.0777),
+            'Ni': (-2.516, 0.0969),
+            'W': (-4.574, 0.1209),
+            'Mo': (-3.05, 0.1094),
+            'H': (-0.137, 0.0307)
         }
```

Comment:
Why did the values change here? Was this just a case of the values in the test file being slightly out of date? Just want to confirm that the main .yaml file was not affected.

Reply:
The values changed since the previous version actually had a mistake. This particular test case tests for whether the CorrectionCalculator works if there is no experimental uncertainty data available (in which case it should compute corrections with all compounds weighted equally). Previously, it still filtered out compounds with large uncertainties since filtering was determined by the warnings dict and not the uncertainty data, but I realized that it makes more sense if it doesn't filter by uncertainty.